File: Nivedita/Array/Sort an array of 0s, 1s and 2s.cpp
class Solution
{
    public:
    void sort012(int a[], int n)
    {
       int l=0,m=0,h=n-1;
       for(int i=0;i<n;i++)
       {
           if(a[m]==0)
           {
               int temp=a[m];
               a[m]=a[l];
               a[l]=temp;
               l++;
               m++;
           }
           else if(a[m]==2)
           {
               int t=a[m];
               a[m]=a[h];
               a[h]=t;
               h--;
           }
           else
           {
               m++;
           }
       }// coode here 
    }
    
};

File: JyotiKumari/Sort an array of 0s, 1s and 2s.py
class Solution:
    def sort012(self,arr,n):
        # code here
        i=0
        c0 = arr.count(0)
        c1 = arr.count(1)
        c2 = arr.count(2)
        while(c0):
            arr[i]=0
            i+=1
            c0-=1
        
        while(c1):
            arr[i]=1
            i+=1
            c1-=1
           
        while(c2):
            arr[i]=2
            i+=1
            c2-=1